Library/Tracking/KSRAnalytics.swift

```diff
@@ -706,14 +705,15 @@ public final class KSRAnalytics {
   public func trackAddOnsContinueButtonClicked(
     project: Project,
     reward: Reward,
+    checkoutData: CheckoutPropertiesData,
     refTag: RefTag?
   ) {
     let props = projectProperties(from: project, loggedInUser: self.loggedInUser)
-      .withAllValuesFrom(pledgeProperties(from: reward))
-      .withAllValuesFrom(contextProperties())
+      .withAllValuesFrom(contextProperties(ctaContext: .addOnsContinue))
```

Comment:
This line is not needed because of `717`.

Reply:
`712` adds `context_cta` but `717` adds `context_page` so they're both needed

Reply:
good point but I also dont see a `context_page` defined for the event so  line 717 can be removed.

Reply:
Yea, that's right.